mflike/MFLike.yaml

```diff
@@ -43,33 +47,32 @@ data:
     # For each of them, you can specify which spectra
     # and scale cuts you'd like to use. If you don't
     # specify anything, the defaults will be used.
-    - experiments: [LAT, LAT]
-      frequencies: [93, 93]
-    - experiments: [LAT, LAT]
-      frequencies: [93, 145]
-    - experiments: [LAT, LAT]
-      frequencies: [93, 225]
-    - experiments: [LAT, LAT]
-      frequencies: [145, 145]
-    - experiments: [LAT, LAT]
-      frequencies: [145, 225]
-    - experiments: [LAT, LAT]
-      frequencies: [225, 225]
+    - experiments: [LAT_93, LAT_93]
+    - experiments: [LAT_93, LAT_145]
+    - experiments: [LAT_93, LAT_225]
+    - experiments: [LAT_145, LAT_145]
+    - experiments: [LAT_145, LAT_225]
+    - experiments: [LAT_225, LAT_225]
 
-# Parameters to handle the band integration:
-# - external_bandpass sets the usage of an external bandpass file
-# - polarized_arrays sets the PA we want to use, e.g. polarized_arrays: ['PA1','PA2']
-# - nsteps sets the number of frequencies used in the integration
-# - bandwidth sets the relative width of the band wrt the central frequency
-#     with bandwidth: 0 no band integration is performed
-#     if bandwidth > 0 , nsteps must be > 1
+# Parameters to build a top-hat band:
+# - nsteps sets the number of frequencies used in the band integration
+# - bandwidth sets the relative width of the band wrt the central frequency 
+#   the central frequency of each band is set from the bands stored in the sacc file
+#     - with nstep: 1, bandwidth must be 0
+#     Dirac delta bandpass, no band integration
+#     useful if you don't want to do band integration 
+#     when the bandpasses in the sacc file are multifrequency
+#     the freq used is the effective frequency from the bandpass
+#     - if nstep > 1,  bandwidth must be > 1
 #     bandwidth can be a list if you want a different width for each band
 #     e.g. bandwidth: [0.3,0.2,0.3] for 3 bands
-band_integration:
-  external_bandpass: false
-  polarized_arrays: false
-  nsteps: 1
-  bandwidth: 0
+# without the top_hat_band block: no top-hat band is built and 
+# bandpasses read from sacc file. Band integration is performed accordingly 
+# (if bandpass in sacc is a single freq, no band integration) 
+# the default is to read bandpasses from file, to build top-hat uncomment this block!
+#top_hat_band:
+#  nsteps: 1
+#  bandwidth: 0
 
 systematics_template:
```

Comment:
The `systematics_template` block is not needed anymore if you do not plan to use it. We can add an explanation and comment this part just like the `top_hat_band` block

Reply:
That's true, in the default case we would not use a leakage template. I can comment that block too